### PR TITLE
Ma/auth

### PR DESCRIPTION
**Pull request by mdacoca**
### Motivation and context

User authorization documentation updated:

- Screenshots
- Procedure descriptions
- Links

### Checklist

- [X] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [X] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: site/content/en/docs/manual/basics/authorization.md [hunk 1]
```diff
@@ -4,40 +4,70 @@ linkTitle: 'Authorization'
...
+Use the following command, to create a superuser:
+
+  ```bash
+    docker exec -it cvat_server bash -ic 'python3 ~/manage.py createsuperuser'
+  ```
+
+Then, use [Django administration panel](http://localhost:8080/admin) panel, to:
 
-- Create / edit / delete users
+- Create / edit/delete users
```
Comment (klakhov) [inline site/content/en/docs/manual/basics/authorization.md:70]: Probably excess change

File: site/content/en/docs/manual/basics/authorization.md [hunk 2]
```diff
@@ -4,40 +4,70 @@ linkTitle: 'Authorization'
...
-  on the login page. Don't forget to modify permissions for the new user in the
-  administration panel. There are several groups (aka roles): admin, user,
-  annotator, observer.
+ ![](/images/image001.jpg)
+
+To register as a non-admin user, do the following:
+
+1. Click **Create an account**.
+
+   ![](/images/image002.jpg)
```
Comment (klakhov) [inline site/content/en/docs/manual/basics/authorization.md:29]: What do you think about adding full image for login form?

Comment (mdacoca) [reply site/content/en/docs/manual/basics/authorization.md:29]: @klakhov good idea, updated

File: site/content/en/docs/manual/basics/authorization.md (hunk 1 above)
Comment (mdacoca) [reply site/content/en/docs/manual/basics/authorization.md:70]: updated

**Review by klakhov — APPROVED** [review]

**Comment by nmanovic** [comment]
@mdacoca , there are several problems with the updated documentation.

- We don't have observer role anymore
- By default, users will have 'user' group
- The screenshot which you added from CVAT.ai. On a local machine, the user will not see Google and GitHub authentication methods by default. Need to find a way to document CVAT.ai and local deployment at the same time.

Please ping me and we can discuss.

**Review by nmanovic — CHANGES_REQUESTED** [review]
@mdacoca , please ask me how to re-write the documentation page.

**Review by chernov-a — APPROVED** [review]
From my point of view, the document looks good.

File: site/content/en/docs/administration/basics/admin-account.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+description: 'A CVAT installation guide to create a superuser.'
+---
+
+This section is for users who whant to be a bit more flexible with CVAT use.
+
+The user you register by default does not have the right to view a list of
+tasks or modify permissions, so you must create a superuser.
+The superuser can use [Django administration panel](http://localhost:8080/admin)
+to assign groups (roles) to other users.
+<br>Available roles are: user (default), admin, worker.
```
Comment (nmanovic) [inline site/content/en/docs/administration/basics/admin-account.md:14]:
```suggestion
The user you register by default does not have full permissions on the instance,
so you must create a superuser.
The superuser can use [Django administration panel](http://localhost:8080/admin)
to assign groups (roles) to other users.
<br>Available roles are: user (default), admin, business, worker.
```

File: site/content/en/docs/administration/basics/admin-account.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+Steps of installation are partly different, depending on the type of operation system (OS).
+
+This section starts with **Create superuser** step that is common for all OS.
+
+### Register as a superuser
+
+In the process of installation you need to create a superuser:
+
+1. In the Git Bash run the following command:
```
Comment (nmanovic) [inline site/content/en/docs/administration/basics/admin-account.md:29]:
```suggestion
1. In a terminal run the following command:
```

File: site/content/en/docs/administration/basics/admin-account.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+In the process of installation you need to create a superuser:
+
+1. In a terminal run the following command:
+
+```bash
+  docker exec -it cvat_server bash -ic 'python3 ~/manage.py createsuperuser'
+```
+
+2. Set up username, email adress, and password.
```
Comment (nmanovic) [inline site/content/en/docs/administration/basics/admin-account.md:35]:
```suggestion
2. Set up username, email address, and password.
```

File: site/content/en/docs/administration/basics/admin-account.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+1. In a terminal run the following command:
+
+```bash
+  docker exec -it cvat_server bash -ic 'python3 ~/manage.py createsuperuser'
+```
+
+2. Set up username, email address, and password.
+3. Go to [`localhost:8080`](http://localhost:8080), and log in with credentials from step 2.
+4. (Optional) Go to [Django administration panel](http://localhost:8080/admin) panel to:
+   - Create / edit/delete users
```
Comment (nmanovic) [inline site/content/en/docs/administration/basics/admin-account.md:38]:
```suggestion
   - Create/edit/delete users
```

File: site/content/en/docs/administration/basics/social-accounts-configuration.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+   version: '3.3'
+   services:
+     cvat_server:
+       environment:
+         USE_ALLAUTH_SOCIAL_ACCOUNTS: 'True'
+         SOCIAL_AUTH_GOOGLE_CLIENT_ID: '<YOUR_GOOGLE_CLIENT_ID>'
+         SOCIAL_AUTH_GOOGLE_CLIENT_SECRET: '<YOUR_GOOGLE_CLIENT_SECRET>'
+   ```
+
+   2. In Git Bash, run the following command:
```
Comment (nmanovic) [inline site/content/en/docs/administration/basics/social-accounts-configuration.md:44]:
```suggestion
   2. In a terminal, run the following command:
```

File: site/content/en/docs/administration/basics/social-accounts-configuration.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+   version: '3.3'
+   services:
+     cvat_server:
+       environment:
+         USE_ALLAUTH_SOCIAL_ACCOUNTS: 'True'
+         SOCIAL_AUTH_GITHUB_CLIENT_ID: '<YOUR_GITHUB_CLIENT_ID>'
+         SOCIAL_AUTH_GITHUB_CLIENT_SECRET: '<YOUR_GITHUB_CLIENT_SECRET>'
+   ```
+
+   2. In Git Bash, run the following command:
```
Comment (nmanovic) [inline site/content/en/docs/administration/basics/social-accounts-configuration.md:74]:
```suggestion
   2. In a terminal, run the following command:
```

File: site/content/en/docs/manual/basics/registration.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
+---
+title: 'Registration'
+linkTitle: 'egistration'
```
Comment (nmanovic) [inline site/content/en/docs/manual/basics/registration.md:3]:
```suggestion
linkTitle: 'registration'
```

File: site/content/en/docs/manual/basics/registration.md [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+See:
+
+- [User registration](#user-registration)
+- [Account access](#account-access)
+
+To creata account or log in, go to the [App CVAT](https://app.cvat.ai/) login page:
+
+![](/images/image001.jpg)
+
+> **Note:** By default authentincation and registration with Google and Git works only for [App CVAT](https://app.cvat.ai/). <br>If you want to use Google and Git authentication on local installation, see [Social auth configuration](/docs/administration/basics/social-accounts-configuration/).
```
Comment (nmanovic) [inline site/content/en/docs/manual/basics/registration.md:30]:
```suggestion
> **Note:** By default authentication and registration with Google and GitHub work only for [App CVAT](https://app.cvat.ai/). <br>If you want to use Google and GitHub authentication on a local installation, see [Social auth configuration](/docs/administration/basics/social-accounts-configuration/).
```

**Review by nmanovic — APPROVED** [review]

**Review by nmanovic — APPROVED** [review]

**Pull request merged**